src/patternfly/components/Table/examples/table-compact-example.hbs

```diff
@@ -19,7 +19,7 @@
       {{#> table-th table-th--attribute='scope="col"'}}
         Numbers
       {{/table-th}}
-      {{#> table-th table-th--attribute='scope="col"' table-th--modifier="pf-m-center"}}
```

Comment:
The `th` doesn't align with the content now. Is that intentional?

<img width="470" alt="screen shot 2019-03-01 at 4 10 02 pm" src="https://user-images.githubusercontent.com/35148959/53669394-91015a00-3c3c-11e9-98b6-9af82c90557e.png">

Reply:
Per https://github.com/patternfly/patternfly-org/pull/732, looks like the content should align with the header.

Reply:
Fixed this now :)